src/components/GQLUtils.ts

```diff
@@ -0,0 +1,22 @@
+export interface UseQueryData<T> {
```

Comment:
There don't appear to be any other references to UseQueryData: 
https://sourcegraph.com/github.com/kobra-dev/Kobra@9687f4ee704ecb30087ca4896fa018b0900baeef/-/blob/src/components/GQLUtils.ts?L1:18#tab=references

Reply:
Should be removed?